guides/common/modules/proc_deploying-a-custom-ssl-certificate-to-smart-proxy-server.adoc

```diff
@@ -37,15 +37,15 @@ If the command is successful, it returns two `{certs-generate}` commands, one of
 ----
 Validation succeeded.
 
-To use them inside a NEW {SmartProxyServer}, run this command:
-  {certs-generate} --foreman-proxy-fqdn "{SmartProxyServer}" \
+To use them inside a NEW ${smart-proxy-capitalized}, run this command:
+  {certs-generate} --foreman-proxy-fqdn "${smart-proxy-capitalized}" \
     --certs-tar "~/{smartproxy-example-com}-certs.tar" \
```

Comment:
If you want to be consistent with the output:
```suggestion
    --certs-tar "~/${smart-proxy-capitalized}-certs.tar" \
```

And the same below

Reply:
Applied and rebased to master.